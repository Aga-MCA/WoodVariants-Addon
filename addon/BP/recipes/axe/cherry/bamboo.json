{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "cherry_bamboo_axe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "SP",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:bamboo_stick"
      },
      "P": {
        "item": "aga_wood_variants:cherry_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:cherry_bamboo_axe"
    }
  }
}