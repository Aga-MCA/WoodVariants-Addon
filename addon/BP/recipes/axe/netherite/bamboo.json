{
  "format_version": "1.17",
  "minecraft:recipe_smithing_transform": {
    "description": {
      "identifier": "netherite_bamboo_axe"
    },
    "tags": [
      "smithing_table"
    ],
    "template": "minecraft:netherite_upgrade_smithing_template",
    "base": "aga_wood_variants:diamond_bamboo_axe",
    "addition": "minecraft:netherite_ingot",
    "result": "aga_wood_variants:netherite_bamboo_axe"
  }
}