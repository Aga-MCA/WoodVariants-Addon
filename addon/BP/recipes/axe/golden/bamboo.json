{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_bamboo_axe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "SP",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:bamboo_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_bamboo_axe"
    }
  }
}